src/v2/System/i18n/i18n.ts

```diff
@@ -0,0 +1,24 @@
+import i18n from "i18next"
+import { initReactI18next } from "react-i18next"
+
+const translations = require("v2/System/locales/en-US/translation.json")
+
+i18n.use(initReactI18next).init({
+  debug: true,
```

Comment:
do we need this `true`? 🤔

Reply:
I thought it would be better to keep it `true` as it is helpful in case we have issues when translations can't be loaded (default value is false). Would it be better to remove it?

Reply:
as long as it doesnt break anything for real user on prod, then lets keep it here. I'm just not sure what this `true` entails, so I was being cautious.

Reply:
Good call!
This is the official definition: https://www.i18next.com/overview/configuration-options

> logs info level to console output. Helps finding issues with loading not working

As I understand, it's just about logs, so I don't think it can break anything. Do you think I could be missing something?

Reply:
makes sense. all good then :)

Reply:
We should only enable this in development, which is as easy as checking `debug: getENV("NODE_ENV") === "development"